src/foam/u2/view/RichChoiceView.js

```diff
@@ -392,7 +400,19 @@ foam.CLASS({
                         .start().hide(!! section.hideIfEmpty && resp[index].value <= 0 || ! section.heading)
                           .addClass(self.myClass('heading'))
                           .add(section.heading)
-                        .end()
+                          .end()
```

Comment:
Fix indentation. This and the code you added below should be indented one level less than it is currently.